docker/suricata/dist/suricata.yaml

```diff
@@ -263,8 +268,7 @@ outputs:
         - snmp
         - sip
         - dhcp:
-            # DHCP logging requires Rust.
-            enabled: no
+            enabled: yes
```

Comment:
Not sure you are interested in DHCP, but I noticed that `rust` is now enabled in T-Pot's `suricata/Dockerfile` and `suricata/Dockerfile.from.source`.